.hintrc

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": [
+    "development"
+  ],
+  "hints": {
+    "typescript-config/consistent-casing": "off"
+  }
+}
```

Comment:
I'm not sure we need this. Whats this doing?

Reply:
There was an error suggesting incorrect letter casing I turned it off.

Reply:
Can you elaborate please. What was the error?